pep257.py

```diff
@@ -54,22 +54,43 @@ def next(obj, default=nothing):
 
 PROJECT_CONFIG = ('setup.cfg', 'tox.ini', '.pep257')
 
-humanize = lambda string: re(r'(.)([A-Z]+)').sub(r'\1 \2', string).lower()
-is_magic = lambda name: name.startswith('__') and name.endswith('__')
-is_ascii = lambda string: all(ord(char) < 128 for char in string)
-is_blank = lambda string: not string.strip()
-leading_space = lambda string: re('\s*').match(string).group()
+
+def humanize(string):
+    return re(r'(.)([A-Z]+)').sub(r'\1 \2', string).lower()
+
+
+def is_magic(name):
+    return name.startswith('__') and name.endswith('__')
+
+
+def is_ascii(string):
+    return all(ord(char) < 128 for char in string)
+
+
+def is_blank(string):
+    return not string.strip()
+
+
+def leading_space(string):
+    return re('\s*').match(string).group()
 
 
 class Value(object):
 
-    __init__ = lambda self, *args: vars(self).update(zip(self._fields, args))
-    __hash__ = lambda self: hash(repr(self))
-    __eq__ = lambda self, other: other and vars(self) == vars(other)
+    def __init__(self, *args):
+        vars(self).update(zip(self._fields, args))
+
+    def __hash__(self):
+        return hash(repr(self))
+
+    def __eq__(self, other):
+        return other and vars(self) == vars(other)
 
     def __repr__(self):
-        format_arg = lambda arg: '{}={!r}'.format(arg, getattr(self, arg))
-        kwargs = ', '.join(format_arg(arg) for arg in self._fields)
+        kwargs = ', '.join(
+            map(lambda arg: '{}={!r}'.format(arg, getattr(self, arg)),
+                self._fields)
+            )
```

Comment:
@jirikuncar I'm pretty sure that'll trigger a line to long error which I think @Nurdok will still object to when formatted properly. Regardless, let me see how that looks after accounting for the default maximum line length.